scripts/mkcloud

```diff
@@ -798,10 +803,16 @@ function prepareinstcrowbar()
     return $?
 }
 
+function scp_install_chef_suse_override()
+{
+    if [ -e "$install_chef_suse_override" ]; then
+        $scp -p "$install_chef_suse_override" root@$adminip:/tmp/
```

Comment:
better copy to root@$adminip:/tmp/install-chef-suse.sh because that is the name to be used later and we cannot be certain that the source name is the right one.